entity-framework/core/providers/sql-server/index.md

```diff
@@ -43,6 +43,10 @@ Starting with EF 9, it's recommended to use `UseAzureSql` and `UseAzureSynapse`
 > [!NOTE]
 > UseAzureSql and UseAzureSynapse methods were introduced in EF Core 9.0.
 
+### EnableRetryOnFailureByDefault
+
+EF 9 introduced `EnableRetryOnFailureByDefault` method that configures the context to default [execution strategy](xref:core/miscellaneous/connection-resiliency) unless it is configured explicitly (i.e. when using DI). To use it, simply call `ConfigureSqlEngine(c => c.EnableRetryOnFailureByDefault())` and later you can use `UseSqlServer` or `UseAzureSql` or `UseAzureSynapse` as usual.
```

Comment:
Mention that it's not necessary to call it with the `UseAzure` methods

Reply:
https://github.com/dotnet/EntityFramework.Docs/pull/5000